images/build/Dockerfiles/buildAndRun.Dockerfile

```diff
@@ -0,0 +1,71 @@
+ARG DEBIAN_FLAVOR
+
+### oryx run-script image
+FROM golang:1.15-${DEBIAN_FLAVOR} as startupScriptGens
+
+# GOPATH is set to "/go" in the base image
+WORKDIR /go/src
+COPY src/startupscriptgenerator/src .
+
+ARG GIT_COMMIT=unspecified
+ARG BUILD_NUMBER=unspecified
+ARG RELEASE_TAG_NAME=unspecified
+ENV GIT_COMMIT=${GIT_COMMIT}
+ENV BUILD_NUMBER=${BUILD_NUMBER}
+ENV RELEASE_TAG_NAME=${RELEASE_TAG_NAME}
+ENV DEBIAN_FLAVOR=${DEBIAN_FLAVOR}
+RUN ./build.sh golang     /opt/startupcmdgen/golang
+
+### oryx build image
+FROM buildpack-deps:${DEBIAN_FLAVOR}-curl
+ARG DEBIAN_FLAVOR
+ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
+
+# docker multi-stage builds
+COPY --from=buildscriptgenerator /opt/ /opt/
+COPY --from=startupScriptGens /opt/startupcmdgen/ /opt/startupcmdgen/
```

Comment:
Just to be clear, for other devs/partners reviewing this PR: this is where the size gain happens, right? Basically, we're starting over from the Oryx build image and just copy what we need from previous steps.

Reply:
This is where the size reduction happens. And correct, we copy what we need from the previous steps :)